_opensearch/query-dsl/term-vs-full-text.md

```diff
@@ -0,0 +1,233 @@
+---
+layout: default
+title: Term-level vs. full-text queries
+parent: Query DSL
+nav_order: 20
+---
+
+# Term-level vs. full-text queries
```

Comment:
I really like that you split this out into its own topic! But it feels like it should be before full-text queries in the nav pane. Maybe give an overview statement, something like 'OpenSearch supports both full-text and term-level queries. The main difference between them is that term-level queries ...'

Reply:
Meant for it to be before full-text queries. Updated.